src/ethereum/vm/gas.py

```diff
@@ -19,6 +19,10 @@
 GAS_STORAGE_SET = U256(20000)
 GAS_STORAGE_UPDATE = U256(5000)
 GAS_STORAGE_CLEAR_REFUND = U256(15000)
+GAS_LOW = U256(5)
+GAS_MID = U256(8)
+GAS_EXP = U256(10)
```

Comment:
```suggestion
GAS_EXPONENTIATION = U256(10)
```

Reply:
I know this isn't how the yellow paper listed it, but `EXP` isn't very descriptive at first glance.